README.md

```diff
@@ -5,5 +5,12 @@ In this assignment we will be transferring a small kaggle model to a professiona
 - poetry for dependency management.
 
 ### How to run
-To run this codee you need to have poetry installed. You can install the packages by running the following command:
-- ```poetry install```
\ No newline at end of file
+To run this codee you need to have poetry installed. You can install the packages by running the following commands:
+- ```poetry install```
+- ```poetry shell```
+To retrieve the data and run the pipeline:
+- ```dvc pull```
+- ```dvc repro```
+To run the code quality metrics:
+- ```pylint ./phishing_detection```
+- ```bandit ./```
```

Comment:
Please change this to ```bandit ./ -r``` and specify it needs to be run from the phishing detection folder. Or ```bandit ./phishing_detection/ -r``` for consistency such that everything can be ran from the root of the folder.